austrakka/utils/options.py

```diff
@@ -428,6 +428,27 @@ def opt_hash_check(**attrs: t.Any):
     )
 
 
+def opt_hash_cache(**attrs: t.Any):
+    defaults = {
+        'help': 'Specify whether to reuse previous file hashes after '
+                'initial check. If --no-hash-cache is specified, all existing '
+                'file hashes will be re-checked for consistency with the server '
+                'on every sync. If --hash-cache is specified, file hashes will '
+                'be checked when they are first downloaded; the successful hashes '
+                'are remembered; future hash checks will compare the server hash '
+                'against the locally memorized values. This optimizes the time '
+                'it takes to hash check large files on repeat runs. When a cache '
+                'miss is detected the file is download again and the new hash is '
+                'memorized locally.'
+    }
+    return _create_option(
+        '--hash-cache/--no-hash-cache',
+        type=bool,
+        default=True,
+        **{**defaults, **attrs}
+    )
```

Comment:
Default is true, name of standalone flag equivalent to false could be --check-all-files ?

Reply:
The other pair was --hash-check / --no-hash-check , --no-hash-check is probably a pretty good name to keep

Reply:
Possible text for --recalculate-hashes

Recalculate local file hashes for previously-downloaded files; do not use cached hashes. This can take a long time to run. If this option is not specified, cached local hashes will be used for previously-downloaded files, but hashes will still be calculated for newly-downloaded files.